tests/Doctrine/Tests/Models/Company/CompanyFlexContract.php

```diff
@@ -67,14 +67,6 @@
 #[ORM\Entity]
 class CompanyFlexContract extends CompanyContract
 {
-    /**
-     * @Id
-     * @GeneratedValue
-     * @Column(type="integer")
-     * @var int
-     */
-    public $id;
```

Comment:
@greg0ire Do you still remember what you intended when you made this change?

https://github.com/doctrine/orm/commit/2c769acf8cf5838c65b4dd90b1966418679177cb#diff-22f2b83fc9f9b4f9e167ff860549dd20c8d2985a82bfaaa3e04e56f04ad0f4c9R70-R76

That now (with the changes in this PR) causes some of the test failures, since the ID column is now a field that is declared twice.

Reply:
Sorry, I wish I had written a better commit message, I don't remember.

Reply:
https://github.com/doctrine/orm/pull/9528#issuecomment-1046209939

This comment shows you had issues with the id missing, so maybe you added it to see if it changes anything? (My guess would be that change was not necessary and you fixed the issue in another way.)

Reply:
I might have forgotten to remove it, yes… I suppose the best way to find out is to remove it.